src/services/buildApi.ts

```diff
@@ -85,9 +85,7 @@ export const getDependencies = ({ id }: IBuildApiData, requestConfig: AxiosReque
  * @param requestConfig - Axios based request config
  */
 export const getBuildMetrics = ({ buildIds }: { buildIds?: Array<string> }, requestConfig: AxiosRequestConfig = {}) => {
-  if (buildIds) {
```

Comment:
I am afraid not, I would prefer to keep the logic here because `buildIds` can be empty unless otherwise refactor the Build Metrics to exclude the circumstance that there are no buildIds.

Reply:
I just realized that Patrik moved it to BuildMetrics.tsx so it can be fine here as no other usage of this function.

Reply:
It was problematic, because if the `buildIds` are not defined, the function returns nothing and therefore type system complains.

Reply:
I will also update the type of params of this function so that `buildIds` have to be defined.